packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -25,6 +27,7 @@ describe('I18n', () => {
   beforeEach(() => {
     translate.mockReset();
     getCurrencySymbol.mockReset();
+    getFallbackLocale.mockReset();
```

Comment:
I think the fact that you are only mocking it out to return the actual results you would get with the real version means it would be better to just use the real function instead. You can do that in the mock by including the real module:

```ts
jest.mock('../utilities', () => {
  ...require.requireActual('../utilities'),
  mockOne: jest.fn(),
});
```